packages/@react-aria/interactions/stories/useInteractOutside.stories.tsx

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
This story demonstrates a bug we currently have, https://github.com/adobe/react-spectrum/issues/1338, that this PR does fix, though we need to evaluate if this is what we want to do.